terraform/modules/hono/README.md

```diff
@@ -44,12 +45,20 @@ No modules.
 | <a name="input_hono_tls_key"></a> [hono\_tls\_key](#input\_hono\_tls\_key) | Content of the hono domain tls Key File | `string` | n/a | yes |
 | <a name="input_hono_tls_key_from_storage"></a> [hono\_tls\_key\_from\_storage](#input\_hono\_tls\_key\_from\_storage) | Content of the hono domain tls Key File from storage bucket | `string` | n/a | yes |
 | <a name="input_hono_trust_store_config_map_name"></a> [hono\_trust\_store\_config\_map\_name](#input\_hono\_trust\_store\_config\_map\_name) | Name of the kubernetes trust store config map for the hono deployments managed by trust-manager. | `string` | n/a | yes |
+| <a name="input_hpa_enabled"></a> [hpa\_enabled](#input\_hpa\_enabled) | Enables the creation of a horizontal pod autoscaler for the MQTT adapter and the device registry. | `bool` | n/a | yes |
+| <a name="input_hpa_maxReplicas_device_registry"></a> [hpa\_maxReplicas\_device\_registry](#input\_hpa\_maxReplicas\_device\_registry) | Maximum number of replicas the device registry horizontal pod autoscaler can scale to. | `number` | n/a | yes |
+| <a name="input_hpa_maxReplicas_mqtt"></a> [hpa\_maxReplicas\_mqtt](#input\_hpa\_maxReplicas\_mqtt) | Maximum number of replicas the MQTT horizontal pod autoscaler can scale to. | `number` | n/a | yes |
+| <a name="input_hpa_metrics_mqtt"></a> [hpa\_metrics\_mqtt](#input\_hpa\_metrics\_mqtt) | Metrics for the MQTT horizontal pod autoscaler as JSON list. | `any` | n/a | yes |
+| <a name="input_hpa_minReplicas_device_registry"></a> [hpa\_minReplicas\_device\_registry](#input\_hpa\_minReplicas\_device\_registry) | Minimum number of replicas the device registry horizontal pod autoscaler can scale to. | `number` | n/a | yes |
```

Comment:
> Minimum number of replicas the device registry horizontal pod autoscaler can scale to.

Cosmetic: `Minimum number of replicas the horizontal pod autoscaler is able to spawn for the device registry`

Reply:
I wrote a new description. Please let me know what you think of it.

Reply:
Looks good!